middlewares/auth.middleware.js

```diff
@@ -86,6 +88,33 @@ async function sendResetPasswordEmailMiddleware(req, res, next) {
     }
 }
 
+/**
+ * Middleware that sends an email to confirm the account for the inputted email address.
```

Comment:
Description should include the fact that this should be used only when accounts are for hackers